src/models/shopping-cart.model.ts

```diff
@@ -8,18 +8,22 @@ import {Entity, model, property} from '@loopback/repository';
 /**
  * Item in a shopping cart
  */
+@model()
```

Comment:
We need to make this a model so this is registered with the openapi spec?

Reply:
Yes, that's what leads to https://github.com/strongloop/loopback-next/pull/1710 too.